README.md

```diff
@@ -1,5 +1,15 @@
 # Introduction 
-Project Coimbra
+Project Coimbra 
+
+# Setup for development
+1. In root folder, run `git submodule init` to clone nested repo (DryWetMidi)
+2. Launch solution ProjectCoimbra.UWP/Project.Coimbra.sln
+3. Create your own certificate for development by  
+    3.1. In Visual Studio, Solution Explorer, open Project.Coimbra (Universal Windows) / Package.appxmanifest   
+    3.2. Go to **Packaging** tab, click **Choose Certificate...**, then click **Create...** to create your own certificate for development  
+4. Replace MIDI reference by  
+    4.1. In Visual Studio, Solution Explorer, open Project.Coimbra (Universal Windows) / References, right click to remove current Microsoft General MIDI DLS for Universal Windows Apps reference (showing exclamation)    
+    4.2. Right click on References > Add Reference..., in the left pane, select Universal Windows > Extensions, and enable **Microsoft General MIDI DLS for Universal Windows Apps***

```

Comment:
Formatting error of extra `*`. Please preview the markdown before pushing. You can easily do this using a number of VS Code extensions